tests/integrations/config/test-default.toml

```diff
@@ -1,7 +1,11 @@
-[readpool.coprocessor]
+[readpool]
+
+[readpool.unified]
```

Comment:
Why the file is changed?

Reply:
Update it to be identical to the current config template.

Reply:
But there is no new configuration entry, why the output adds new `readpool` and `readpool.unified` tables?

Reply:
This file is only read and compared with `TiKvConfig::default()`. We don't deserialize the config and compare with this file.
That is, this change is just a late addition. I think this file should be synchronized with `etc/config-template.toml`.